application/src/main/java/org/opentripplanner/transit/service/TimetableRepository.java

```diff
@@ -434,7 +435,7 @@ public Collection<PathTransfer> getTransfersByStop(StopLocation stop) {
     return transfersByStop.get(stop);
   }
 
-  public Collection<PathTransfer> getTransfersByMode(StreetMode mode) {
+  public List<PathTransfer> getTransfersByMode(StreetMode mode) {
```

Comment:
I think this method should be called `findTransfers` according to the naming conventions.

Reply:
I renamed `getTransfersByMode` to `findTransfers`